src/Microsoft.Sbom.Targets/SbomCLIToolTask.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Targets;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+/// <summary>
+/// MSBuild ToolTask for generating an SBOM using the SBOM CLI tool
+/// </summary>
+public partial class GenerateSbom : ToolTask
+{
+    /// <summary>
+    /// Path to the SBOM CLI tool
+    /// </summary>
+    [Required]
+    public string SbomToolPath { get; set; }
+
+    protected override string ToolName => "Microsoft.Sbom.Tool";
```

Comment:
Comment

Reply:
Moved this property to GenerateSbom.cs and updated the comment!